sympy/concrete/summations.py

```diff
@@ -523,6 +524,11 @@ def _dirichlet_test(g_n):
                 dirich2 = _dirichlet_test(a_n)
                 if dirich2 is not None:
                     return dirich2
+            ### --------- Test bounded factor ------------###
+            if ask(Q.finite(a_n)) and Sum(b_n, (sym, interval.inf, interval.sup)).is_convergent():
```

Comment:
You need to check for absolute convergence here, not just convergence.  When you fix this, you should add a unit test to make sure something like `Sum((-1)**n * (-1)**n/n, (n, 1, oo))` returns not convergent.